### ISSUE TITLE
Error generating bundle

### ISSUE THREAD

**Issue by timhaywood** [hyperbrew/vite-cep-plugin#9]
Related to #8 

On build Vite exits during generating the bundle, due to an error in the plugin.

```
[cep] Cannot read property 'replace' of undefined
error during build:
TypeError: Cannot read property 'replace' of undefined
```

For me it was line [254](https://github.com/hyperbrew/vite-cep-plugin/blob/05508d0d11d29ba9175ab57eb7e7ef15b1330391/src/index.ts#L254) where it's replacing fixing the `/assets/` paths in `.js` files.

I haven't dove in enough to understand the full context, but it seems that some `.js` files don't contain a `code` property.
These are the available keys of the file that it errors on:

```ts
[ 'fileName', 'isAsset', 'name', 'source', 'type' ];
```

Simply checking `.code` exists worked for me:

```ts
      if (bundle[jsFilename].code) {
        // fix paths
        bundle[jsFileName].code = bundle[jsFileName].code.replace(
          /(\/assets\/)/g,
          "../assets/"
        );
      }
```

Created draft fix #10

**Comment by justintaylor-dev** [comment]
Thanks! Fixed in 0.1.6 and onward

**Comment by timhaywood** [comment]
Amazing, thanks Justin!
